src/main/java/com/github/madhavdhatrak/blaze4j/Blaze4jLogger.java

```diff
@@ -0,0 +1,41 @@
+package com.github.madhavdhatrak.blaze4j;
+
+import java.util.logging.ConsoleHandler;
+import java.util.logging.Handler;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * Utility class for configuring logging in Blaze4j.
+ */
+public class Blaze4jLogger {
+    private static final Logger LOGGER = Logger.getLogger(Blaze4jLogger.class.getName());
+    
+    /**
+     * Configure global logging for all Blaze4j components with a single call.
+     * This will set up both the specific loggers and the root logger to ensure
+     * messages at the specified level are visible.
+     * 
+     * @param level The java.util.logging.Level to set (e.g., Level.INFO, Level.FINE)
+     */
+    public static void configureLogging(Level level) {
```

Comment:
I'm not 100% sure what is the point of this class. It is used only from the test, isn't it?